src/components/SearchPage.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div id="search_page">
+    <custom-header />
+    <result-control
+      :number_of_movies="cards.length"
+      :first_option="realesed_date_text"
+      :first_option_value="realesed_date_value"
+      :second_option="rating_text"
+      :second_option_value="rating_value"
+      :options_name="sort_by"
+    />
+    <film-gallery :cards="cards" />
+    <custom-footer :company_name="company_name" :app_name="app_name" />
+  </div>
+</template>
+
+<script>
+import CustomHeader from "./CustomHeader.vue";
+import CustomFooter from "./CustomFooter.vue";
+import FilmGallery from "./FilmGallery.vue";
+import ResultControl from "./ResultControl.vue";
+import { EventBus } from "../event-bus";
+import { PROGRAM_DATA, I18N, APP_DATA, MOCK_DATA } from "../core/constants";
+export default {
+  name: "SearchPage",
+  components: {
+    CustomHeader,
+    CustomFooter,
+    ResultControl,
+    FilmGallery
+  },
+  data() {
+    return {
+      realesed_date_text: I18N["EN"].REALEASED_DATE,
```

Comment:
For the sake of consistency, consider making all other I18N access this way.
Do keep in mind that `I18N['EN']` is dynamic property access and it allows you to hook some language changing logic up, whereas `I18N.EN` is static property access that can't be change in runtime.